concourse/pipelines/deployment-kick-off.yml

```diff
@@ -82,3 +82,32 @@ jobs:
                 echo "Pipeline kick-off is enabled. Updating. (set ENABLE_MORNING_DEPLOYMENT=false to disable)"
 
                 make -C ./paas-cf "${AWS_ACCOUNT}" trigger-deploy
+
+  - name: pause-kick-off
+    serial: true
+    plan:
+      - get: deployment-timer
+        trigger: true
+      - get: paas-cf
+      - task: pause-kick-off
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/self-update-pipelines
+              tag: 0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2
+          inputs:
+            - name: paas-cf
+          params:
+            AWS_ACCOUNT: ((aws_account))
+            AWS_DEFAULT_REGION: ((aws_region))
+            DEPLOY_ENV: ((deploy_env))
+            SKIP_AWS_CREDENTIAL_VALIDATION: true
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                make -C ./paas-cf "${AWS_ACCOUNT}" pause-kick-off
```

Comment:
We have a variable `((makefile_env_target))` which we use in situations like this. It is used precisely for this purpose: differentiating `make` targets that use the same AWS account (for example, prod and prod-lon). I know we don't use this deployment kick-off in prod, but I think we should be consistent.